### PR TITLE
Fix: Update ESLint to handle class static member 

### PR DESCRIPTION
**Pull request by chakflying**
Having static member in a class is only supported in ES7+ (I think). Bump the version in ESlint will clear the error in `database.js`.

### CODE REVIEW

**Comment by Saibamen** [comment]
Parsing error is gone for me (Visual Studio Code), but I still cannot see other errors and warnings in `database.js` (for example when I change `let` to `var`

**Comment by chakflying** [comment]
Big oops, I basically set an invalid env and made ESLint crash. Should be fixed now, but I has to pull in `babel-eslint`.

**Review by Saibamen — APPROVED** [review]
Works for me. Ready to merge

**Comment by Saibamen** [comment]
@chakflying: Could you also fix remaining files? If you are using VS Code, here is how to auto-fix files during file save: https://github.com/louislam/uptime-kuma/pull/113#issue-696595568

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
         "@vitejs/plugin-legacy": "^1.5.0",
         "@vitejs/plugin-vue": "^1.3.0",
         "@vue/compiler-sfc": "^3.1.5",
+        "babel-eslint": "^10.1.0",
```
Comment (louislam) [inline package.json:62]: Are we using the right package? This package has been deprecated according to https://www.npmjs.com/package/babel-eslint.

Comment (chakflying) [reply package.json]: Switched to new babel-eslint-parser. I guess I won't pull in babel, since we aren't using experimental syntax (?)

**Comment by chakflying** [comment]
`database.js` fixed. Rebased on master.

**Comment by chakflying** [comment]
BTW is semicolon turned off on purpose? Currently we have it all over the place, random lines have or have no semicolon. IMO consistency on this would be better. I also find lines with semi more readable.

**Comment by louislam** [comment]
Lol, just saw another eslint error.

The reason why I disable eslint in the first place is, I don't want to spend time to solve these eslint error...

image

**Comment by chakflying** [comment]
@louislam 
Ooops, didn't know vue also has [its own parser](https://eslint.vuejs.org/user-guide/#how-to-use-a-custom-parser). Fix by replacing `parser` and `parserOptions`:

```js
    parser: "vue-eslint-parser",
    parserOptions: {
        parser: "@babel/eslint-parser",
        sourceType: "module",
        requireConfigFile: false,
    },
```

It really isn't ESLint's problem, the pain comes from rarely having to set this up myself :( The blessed cli generating starter projects has spoiled us...

**Comment by louislam** [comment]
> ```js
> vue-eslint-parser
> ```



> @louislam
> Ooops, didn't know vue also has [its own parser](https://eslint.vuejs.org/user-guide/#how-to-use-a-custom-parser). Fix by replacing `parser` and `parserOptions`:
> 
> ```js
>     parser: "vue-eslint-parser",
>     parserOptions: {
>         parser: "@babel/eslint-parser",
>         sourceType: "module",
>         requireConfigFile: false,
>     },
> ```
> 
> It really isn't ESLint's problem, the pain comes from rarely having to set this up myself :( The blessed cli generating starter projects has spoiled us...

Thank you for your investigation. You are right, I skipped eslint when I was creating this project via vite.js cli. 😂

However, since I put the server part in the same project, hopefully it won't be a problem for backend code.

**Pull request merged**